llmfoundry/utils/builders.py

```diff
@@ -497,10 +497,10 @@ def _validate_cfg(icl_cfg: DictConfig):
             icl_cfg.batch_size = default_batch_size
         if 'pass_at_k' not in icl_cfg:
             icl_cfg.pass_at_k = 1
-        if 'num_beams' not in icl_cfg:
```

Comment:
Should we error if num_beams is specified # outside of generation kwargs? I think yes because people will probably still have yamls with that

Reply:
raised an error explaining the remapping